proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -31,47 +31,72 @@ import "google/protobuf/timestamp.proto";
 import "temporal/api/deployment/v1/message.proto";
 import "temporal/api/common/v1/message.proto";
 
+// Data for each deployment+task queue pair. This is stored in each deployment (for each task
+// queue), and synced to task queue user data (for each deployment).
+message Data {
+    google.protobuf.Timestamp first_poller_time = 1;
+    google.protobuf.Timestamp last_became_current_time = 2;
+}
+
 message DeploymentLocalState {
     temporal.api.deployment.v1.Deployment worker_deployment = 1;
     map<string, TaskQueueFamilyInfo> task_queue_families = 2;
     bool is_current = 3;
     google.protobuf.Timestamp create_time = 4;
-    map<string, temporal.api.common.v1.Payload> metadata = 5; 
-    
+    map<string, temporal.api.common.v1.Payload> metadata = 5;
+
     message TaskQueueFamilyInfo {
         // key: taskQueueType, val: TaskQueueInfo
-        map<int32,TaskQueueInfo> task_queues = 1;
-        
+        map<int32, TaskQueueInfo> task_queues = 1;
+
         message TaskQueueInfo {
             temporal.api.enums.v1.TaskQueueType task_queue_type = 1;
-            google.protobuf.Timestamp first_poller_time = 2;
+            Data data = 2;
         }
     }
 }
 
+// used as deployment workflow input:
 message DeploymentWorkflowArgs {
     string namespace_name = 1;
     string namespace_id = 2;
     DeploymentLocalState deployment_local_state = 3;
 }
 
+// used as deployment series workflow input:
 message DeploymentSeriesWorkflowArgs {
     string namespace_name = 1;
     string namespace_id = 2;
     string default_build_id = 3;
 }
 
-message RegisterWorkerInDeploymentArgs{
+// used as deployment workflow update input:
+message RegisterWorkerInDeploymentArgs {
     string task_queue_name = 1;
     temporal.api.enums.v1.TaskQueueType task_queue_type = 2;
     google.protobuf.Timestamp first_poller_time = 3;
 }
 
+// TODO: comment what this is used as
```

Comment:
this is used as a response when deployment read API queries the deployment workflow, enquiring it's status. It should maybe be called `QueryDescribeDeploymentResponse`